include/cantera/kinetics/solveSP.h

```diff
@@ -486,5 +489,8 @@ class solveSP
 public:
     int m_ioflag = 0;
 };
+
+//! @}
```

Comment:
I'm not quite sure why, but Doxygen is giving me the warning:
```/Users/speth/src/cantera/include/cantera/kinetics/solveSP.h:497: warning: end of file while inside a group```

Reply:
I pointed my doxygen to 1.9.1, and don't get warnings. I am counting 3 instances of `//! @{` and `//! @}`, respectively.

Reply:
Interesting. I'm running 1.9.6 locally, but as you say, the delimiters look correct and balanced, so I'm not sure what's driving this warning. I'm not too concerned about it.